Modules/vc-module-pricing/VirtoCommerce.PricingModule.Test/ExportImport2Test.cs

```diff
@@ -242,7 +286,7 @@ public Task PriceCsvExport()
 
             var metadata = ExportedTypeMetadata.GetFromType<Price>(true);
             resolver.ResolveExportedTypeDefinition(typeof(Price).Name)
-                .WithDataSourceFactory(dataQuery => new PriceExportPagedDataSource(searchServiceMock.Object, priceServiceMock.Object) { DataQuery = dataQuery })
+                .WithDataSourceFactory(dataQuery => new PriceExportPagedDataSource(searchServiceMock.Object, priceServiceMock.Object, authorizationPolicyProviderMock.Object, authorizationServiceMock.Object, userClaimsPrincipalFactoryMock.Object, userManagerMock.Object) { DataQuery = dataQuery })
```

Comment:
Could you add test that ensures that exception is thrown when authorization fails?